### DISCUSSION TITLE
Language switch keyboard shortcut in BO

### DISCUSSION THREAD

**Discussion by fall-en** [PrestaShop/PrestaShop#40162] [Ideas]
#### Is your feature request related to a problem?

Switching between languages in the BO Product Page (or in general in all BO areas where language switch is required) to write description, title, ecc of a single product take away few clicks. I am used to write each part of a product in each language before switching to the next part. Example first compile the title in all languages, after that the summary, etc. Every time I have to  switch language manually by moving my mouse over the switcher near the title.

#### Describe the solution you'd like

Would be nice to have a keyboard shortcut to switch beetween languages in the BO Product page.

#### Alternatives you've considered

In BO show language switch dropdown menu as simple buttons if languages are <= 3, and a dropdown menu if > 3

**Comment by florine2623** [comment]
Hello @fall-en ,

Thank you for your suggestion. Please share what king of shortcut you would like to use. If they are any screenshots you can also share them.
The Product Team will take it into consideration for future developments. 

Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you create it. If you are willing to implement this feature into PrestaShop as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/), the teams will be happy to help you by ensuring that it is appropriate and coherent with the rest of the project. If that is the case, please let us know!

Thank you

**Comment by matks** [comment]
ping @MatShir 😉

**Comment by fall-en** [comment]
Hello @florine2623 
I have made a mockup of two cases:

1. If there are only two languages would be easy to have two buttons and a keyboard shortcut to switch between them. ALT+SHIFT+ARROW UP or ALT+SHIFT+ARROW DOWN to easily switch between both of them.
![language-selector-1](https://user-images.githubusercontent.com/48456655/123290614-b606ee80-d511-11eb-91b7-3ac77c2bdc84.png)
2. If there are more then two languages the selector should change in a dropdown menu (like current one). ALT+SHIFT+ARROW UP or ALT+SHIFT+ARROW DOWN to easily switch between all of them.
![language-selector-2](https://user-images.githubusercontent.com/48456655/123291004-01b99800-d512-11eb-9897-00a93c9237df.png)
